test.md

```diff
@@ -0,0 +1 @@
+"This file should be ignored by git".
```

Comment:
- [ ] Based on project instructions, this particular file should be ignored by git. Kindly check my review in the `.igignore` file to edit the file name. The instruction was given to show you how ignored files are not meant to appear in your repo 👍 

## Instruction:
![image](https://user-images.githubusercontent.com/50990925/226791841-fc0f4cd4-142f-44ec-8c96-704d997d99d7.png)